docker/Dockerfile

```diff
@@ -70,11 +70,12 @@
 
 
 ARG ARTIFACT_BASE=source-clean
+ARG MAKE_BUILD=make-build
 
 ################################################################################
 # Image containing the run-time dependencies for Sage                          #
 ################################################################################
-FROM ubuntu:jammy as run-time-dependencies
+FROM ubuntu:latest as run-time-dependencies
```

Comment:
I would not use `latest` here. In the past things usually broke when ubuntu upgraded their "latest" tag.

Reply:
Thanks! I will change that in the next commit!